{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -288,13 +282,13 @@ def _env_get_required(setting_name):
 
 PRIVATE_MEDIAFILES_LOCATION = ""
 # Django Storages configuration
-if os.environ.get("USE_AWS_STORAGE", "False") == "True":
-    AWS_ACCESS_KEY_ID = _env_get_required("AWS_ACCESS_KEY_ID")
-    AWS_STORAGE_BUCKET_NAME = _env_get_required("AWS_STORAGE_BUCKET_NAME")
-    AWS_SECRET_ACCESS_KEY = _env_get_required("AWS_SECRET_ACCESS_KEY")
+if config("USE_AWS_STORAGE", default=False):
+    AWS_ACCESS_KEY_ID = config("AWS_ACCESS_KEY_ID")
+    AWS_STORAGE_BUCKET_NAME = config("AWS_STORAGE_BUCKET_NAME")
+    AWS_SECRET_ACCESS_KEY = config("AWS_SECRET_ACCESS_KEY")
     AWS_S3_CUSTOM_DOMAIN = AWS_STORAGE_BUCKET_NAME + ".s3.amazonaws.com"
-    AWS_LOCATION = os.environ.get("AWS_LOCATION", "")
-    AWS_S3_REGION_NAME = _env_get_required("AWS_S3_REGION_NAME")
+    AWS_LOCATION = config("AWS_LOCATION", default="")
+    AWS_S3_REGION_NAME = config("AWS_S3_REGION_NAME")
```

Comment:
@oudeismetis - Out of curiousity, what sort of error is thrown for missing env vars?

Reply:
I did test this and it throws an exception, but I don't have it in my console anymore.
Can't find an example in their docs...

Reply:
https://github.com/henriquebastos/python-decouple/blob/master/decouple.py#L90
I believe this was what I saw in the server logs

Reply:
Ah, just a custom exception. Cool!